includes/managers/class-fs-admin-menu-manager.php

```diff
@@ -360,9 +360,16 @@ function get_top_level_menu_slug() {
 		 * @return bool
 		 */
 		function is_activation_page() {
-			return isset( $_GET['page'] ) &&
-			       ( ( strtolower( $this->_menu_slug ) === strtolower( $_GET['page'] ) ) ||
-			         ( strtolower( $this->_plugin_slug ) === strtolower( $_GET['page'] ) ) );
+			$fs = freemius( $this->_plugin_slug );
+
+			if ( $fs->get_plugin()->is_plugin() ) {
+				return isset( $_GET['page'] ) &&
+				       ( ( strtolower( $this->_menu_slug ) === strtolower( $_GET['page'] ) ) ||
+				         ( strtolower( $this->_plugin_slug ) === strtolower( $_GET['page'] ) ) );
+			}
+
+			global $pagenow;
```

Comment:
@fajardoleo as mentioned in one of the comments above, when there's an admin menu item params set with the init SDK, then this function should use the same logic and return `true` based on the similar logic as with plugin type.